drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -68,6 +68,87 @@ local device_type_profile_map = {
 }
 local detect_matter_thing
 
+local embedded_cluster_utils = require "embedded-cluster-utils"
+
+local CUMULATIVE_ENERGY_IS_REPORTED = "CUMULATIVE_ENERGY_IS_REPORTED"
+local TOTAL_EXPORTED_ENERGY = "TOTAL_EXPORTED_ENERGY"
+local LATEST_EXPORTED_TIMESTAMP = "LATEST_EXPORTED_TIMESTAMP"
+local RECURRING_EXPORT_REPORT_POLL_TIMER = "RECURRING_EXPORT_REPORT_POLL_TIMER"
+
+local REPORT_SCHEDULE_INIT_CALLED = "REPORT_SCHEDULE_INIT_CALLED"
+local REPORT_POLL_TIMER_IS_SET = "REPORT_POLL_TIMER_IS_SET"
+local RECURRING_POLL_TIMER = "RECURRING_POLL_TIMER"
+local REPORT_TIMEOUT = "REPORT_TIMEOUT"
+local MINIMUM_ST_ENERGY_REPORT_INTERVAL = (15 * 60) -- 15 minutes, reported in seconds
+local SUBSCRIPTIONS_COMPLETE = "SUBSCRIPTIONS_COMPLETE"
```

Comment:
What is the purpose of this flag? I'm not sure it is needed, but if it is a comment explaining why would be helpful

Reply:
Related to another comment I left :+1: